src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+import Book from './Book';
+import Form from './form';
+import Navbar from './Navbar';
+
+function Books() {
+  const bookList = useState([{ title: '', author: '' }]);
+  return (
+    <div>
+      <Navbar />
+      {bookList.forEach((book) => (<Book title={book.title} author={book.author} />))}
```

Comment:
- [ ] Greate work implementing this functionality however this component should render a list of books using the Book component, kindly render the books (you can hard code the list of books for now)

Reply:
@Ismailco  Sorry bro. I couldn't understand. In the project requirements, they said about this component that it "Should display the list of books (**empty at this point** but it should be ready for the data) with the Remove button (no functionality yet) ". 
That's why I keep it empty otherwise I could hard code books. 
Thanks, brother.

Reply:
if you got stuck you can use Array.map with the key for each book, something like this : 
```
{bookList.map((book) => (
        <Book key={book.title} title={book.title} author={book.author} />
      ))}
```

Reply:
@Ismailco Oh sorry. I was using the forEach instead of map. Thank you so much. Hey, bro can you help me with a personal thing I will send you a message if you can, please?

Reply:
On slack